build.gradle

```diff
@@ -29,5 +30,31 @@ test.finalizedBy("checkstyleMain")
 
 task runHelloWorldJoseG (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.helloworld.joseg.Main'
+    standardInput=System.in
     classpath = sourceSets.main.runtimeClasspath
-}
\ No newline at end of file
+}
+
+task runferHighlow (dependsOn: 'classes', type: JavaExec) {
+    main = 'jalau.at18.katas.highestlowest.fernanda.Main'
+    standardInput=System.in
+    classpath = sourceSets.main.runtimeClasspath
+
+    
```

Comment:
try to avoid unnecessary empty line as much as possible, they can make the code harder to read

Reply:
Empty lines removed